CMakeLists.txt

```diff
@@ -1,11 +1,7 @@
 # CMake build script for ZeroMQ
-project(ZeroMQ)
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
2.8.12

Reply:
I have pushed the new commit into the patch with the lower version required (2.8.12).